system/jlib/jlog.cpp

```diff
@@ -327,6 +333,32 @@ const LogMsgJobInfo & checkDefaultJobInfo(const LogMsgJobInfo & _jobInfo)
     return _jobInfo;
 }
 
+const LogMsgTraceInfo & queryActiveTraceInfo()
+{
+    if (defaultTraceInfoId != UnknownTraceInfoId)
+    {
+        const LogMsgTraceInfo * traceInfo = theManager->queryTraceInfo(defaultTraceInfoId);
+        if (traceInfo)
+            return *traceInfo;
+    }
+
+    return unknownTrace;
+}
+
+LogMsgTraceInfoId setDefaultTraceInfo(const char *theTraceID, const char *theSpanID, bool threaded)
+{
+    return setDefaultTraceInfo(theManager->addTraceInfo(theTraceID, theSpanID), threaded);
+}
+
+LogMsgTraceInfoId setDefaultTraceInfo(LogMsgTraceInfoId id, bool threaded)
```

Comment:
cleanup: The threaded parameter can be removed since globalDefaultTraceInfoId is not needed.